### ISSUE TITLE
Fix DependencyInjection tests

### ISSUE THREAD

**Issue by javiereguiluz** [symfony/symfony#17594]
Our test suite is broken because of this error reported by Travis for PHP 5.6 and `deps=high` ([example](https://travis-ci.org/symfony/symfony/jobs/105424685#L2427))

```
Fatal error:
Call to undefined method Symfony\Component\DependencyInjection\Definition::getParent() 
in /home/travis/build/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Tests/
DependencyInjection/FrameworkExtensionTest.php on line 535

KO FrameworkBundle
```

**Comment by paradajozsef** [comment]
I have a solution here for this error message: #17582 

But there are another failing tests, so it's not finished yet.

**Comment by paradajozsef** [comment]
Tests has been fixed and merged in these PRs: #17605 #17582.
